check_diff/tests/bash_commands.rs

```diff
@@ -0,0 +1,12 @@
+use check_diff::change_directory_to_path;
+use std::env;
+use tempfile::Builder;
+
+#[test]
+fn cd_test() {
+    // Creates an empty directory in the current working directory
+    let dir = Builder::new().tempdir_in("").unwrap();
+    let dest_path = dir.path();
+    let _ = change_directory_to_path(dest_path).unwrap();
```

Comment:
`change_directory_to_path` returns `io::Result<()>`. No need to bind this to anything if you're just unwrapping:
```suggestion
    change_directory_to_path(dest_path).unwrap();
```